org.kde.kate.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.kate",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "kate",
+    "rename-icon": "kate",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=cups",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host"
+    ],
+    "separate-locales": false,
```

Comment:
Why not?

Reply:
Good question, i copied it from somewhere but makes no sense, i've removed it